### PR TITLE
Bug bash for 489, 491, 493, 494, 497

### PR DESCRIPTION
**Pull request by Haocen**
Fix click on new paste on clone paste editing view not removing custom attachment

Fix cloning paste with attachment

Update CSP in sample and default configuration

Ensure clone paste also clone format

Fix clone button hiding logic when paste is burn after read

Remove attachment name when new paste clicked on

Update SRI

This PR fixes 

## Changes

https://github.com/PrivateBin/PrivateBin/issues/494
https://github.com/PrivateBin/PrivateBin/issues/493
https://github.com/PrivateBin/PrivateBin/issues/491
https://github.com/PrivateBin/PrivateBin/issues/489
https://github.com/PrivateBin/PrivateBin/issues/497

## ToDo

* [ ] 
* [ ] 
* [ ]

### CODE REVIEW

File: js/privatebin.js [hunk 1]
```diff
@@ -2225,6 +2221,18 @@ jQuery.PrivateBin = (function($, RawDeflate) {
...
         };
 
+        /**
+         * returns if status is editing
+         *
+         * @name   Editor.isEditing
+         * @function
+         * @return {bool}
+         */
+        me.isEditing = function()
```
Comment (elrido) [inline js/privatebin.js]: I noticed that in all cases that you call this you negate the result. Hope you don't mind, but I'll remove the "not" from this functions return all the calls of it and rename it accordingly (i.e. Editor.isHidden), so the logic is a bit clearer.

Comment (Haocen) [reply js/privatebin.js]: I'm also looking into this just then, I believe this function and its usage introduce some undesired assumption, e.g. attachment is editable only when textarea visible, I propose to change it to use following function instead:
```
       /**
         * returns if attachment dropdown is editable
         * 
         * @name   TopNav.isAttachmentEditable
         * @function
         * @return {bool}
         */
        me.isAttachmentEditable = function()
        {
            return !$attach.hasClass('hidden');
        }
```

**Comment by elrido** [comment]
I dub thee: Haocen, slayer of bugs :ant::hammer:   <- _no ants were hurt in the production of this emoticon_

File: js/privatebin.js (hunk 1 above)
Comment (rugk) [reply js/privatebin.js]: @elrido Will we still change that last thing @Haocen mentioned?

Comment (Haocen) [reply js/privatebin.js]: I'm working on a seperate PR right now, sorry for the delay.

**Comment by rugk** [comment]
BTW, next time, you can (automatically) [let issues close](https://help.github.com/articles/closing-issues-via-commit-messages/) when a PR is merged. But you need to write this _before each_ issue number.

**Comment by Haocen** [comment]
Cool tip, thanks!

File: js/privatebin.js (hunk 1 above)
Comment (elrido) [reply js/privatebin.js]: Sorry, I didn't keep up with reading the updates. Let's address this in that other PR then. Sorry again for the hassle.

**Pull request merged**